_rules/visible-label-in-accessible-name-2ee8b8.md

```diff
@@ -118,61 +129,273 @@ This `button` element has the text "search" rendered as an magnifying glass icon
+	<p>Hello</p>
+	<p>world</p>
+</button>
+```
+
+#### Passed Example 8
+
+Similar to the previous example.
+
+```html
+<a href="#" aria-label="Some article by John Doe"
+	><h6>Some article</h6>
+	<p>by John Doe</p></a
+>
+```
+
+#### Passed Example 9
+
+The [visible inner text][] of this link is "ACT" (with no spaces) because of the explicit styles of `display: inline` on the `p` elements and the absence of whitespace between the `p` elements. The cases of `display: inline` and `display: block` are handled by the definition of [visible inner text of an element][]. This example shows that the definition agrees with the visual rendering done by the browser.
+
+```html
+<a href="#" aria-label="ACT">
+	<p style="display: inline">A</p>
+	<p style="display: inline">C</p>
+	<p style="display: inline">T</p>
+</a>
+```
+
+#### Passed Example 10
+
+The [visible inner text][] is "Download specification". The words "the" and "gizmo" aren't part of it.
+
+```html
+<a aria-label="Download specification" href="#"
+	>Download <span style="visibility: hidden">the</span> <span style="display: none">gizmo</span> specification</a
+>
+```
+
+#### Passed Example 11
+
+The [visible inner text][] is "Download specification", which includes a space character between the two words due to the second clause of the definition of [visible inner text of a text node][].
+
+```html
+<a aria-label="Download specification" href="#"
+	><span>Download</span><span id="space"> </span><span>specification</span></a
+>
+```
+
+#### Passed Example 12
+
+This example shows that the [visible inner text][] isn't always the same as the [innerText][https://html.spec.whatwg.org/multipage/dom.html#the-innertext-idl-attribute]. The visible inner text is "Download specification". The innerText is 'Download \ngizmo\nspecification'. This rule uses the visible inner text - not innerText.
+
+```html
+<style>
+	.visually-hidden {
+		/* Source: https://www.tpgi.com/the-anatomy-of-visually-hidden/ */
+		clip-path: inset(50%);
+		height: 1px;
+		overflow: hidden;
+		position: absolute;
+		white-space: nowrap;
+		width: 1px;
+	}
+</style>
+<a aria-label="Download specification" href="#">Download <span class="visually-hidden">gizmo</span> specification</a>
+```
+
+#### Passed Example 13
+
+This example shows that the [label in name algorithm][] handles many kinds of whitespace.
+
+```html
+<a aria-label="compose email" href="#"
+	>compose &nbsp;&nbsp;<br />
+	email</a
+>
+```
+
+#### Passed Example 14
+
+This example passes the rule because "YYYY-MM-DD" is in brackets. Text in brackets is removed by the [label in name algorithm][], because its not normally spoken by speech-input users.
+
+```html
+<button aria-label="Search by date">Search by date (YYYY-MM-DD)</button>
+```
+
+#### Passed Example 15
+
+The passes for two reasons: 1) because the ellipsis ("…") is [non-text content][], and 2) because the ellipsis is neither a letter nor a digit and so is filtered out by the [label in name algorithm][].
```

Comment:
```suggestion
This passes for two reasons: 1) because the ellipsis ("…") is [non-text content][], and 2) because the ellipsis is neither a letter nor a digit and so is filtered out by the [label in name algorithm][].
```